docs/topics/Disaster-Recovery.md

```diff
@@ -16,14 +16,28 @@ Backups are done with `pg_dump` and restores are done with `psql` ([see here](ht
 
 ### Backup the database
```

Comment:
This section probably should probably be moved outside of a disaster recovery topic. Don't worry about it for this PR, just making a note.